### PR TITLE
Add comment disclaimer to all resources

### PR DESCRIPTION
**Pull request by steventlamb**
connects #2172 on github

### CODE REVIEW

File: opentreemap/treemap/templates/treemap/map_feature_detail.html [hunk 1]
```diff
@@ -193,13 +193,21 @@ <h4 class="address" id="map-feature-address">{{ feature.address_full }}</h4>
             {% endif %}
           </div>
 
-          {% if feature.is_plot %}
           <div id="comment_disclaimer">
             <p class="text-muted">
-              <em>The comment system does not serve as a way to report problems with a tree.</em>
+              <em>
+                {% if feature.is_plot %}
+                {% blocktrans with resource=term.resource %}
```
Comment (jwalgran) [inline opentreemap/treemap/templates/treemap/map_feature_detail.html]: We can drop the unused "with" variable in this first ``blocktrans``.

**Comment by jwalgran** [comment]
This does show disclaimers for resources, but there is problem with a/an agreement when using terminology that starts with a vowel. I don't know if it is an edge case worth worrying about.

screen shot 2015-07-29 at 3 06 12 pm

**Comment by jwalgran** [comment]
+1 tested

**Comment by steventlamb** [comment]
@jwalgran @RickMohr it looks like the problem of 'a' vs 'an' would not be exclusive to this new text:
https://github.com/OpenTreeMap/otm-core/blob/master/opentreemap/opentreemap/context_processors.py#L60
for example:
```
             data-action='addresource'>{% trans "Add a" %} {{ term.Resource }}</a>
```

Should we revisit this with a new issue?

**Comment by jwalgran** [comment]
:+1:  to a new issue.

**Pull request merged**